### DISCUSSION TITLE
good default value for start_pref_v, limit_pref_v

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/deepmd-kit#1265] [Q&A] [answered]
Hi All,

I want to train a dataset by fitting Energies, Forces, and Virials but I don't have any idea about good value for start_pref_v and limit_pref_v. There is a default value for energy and force but not for virial, so Could you give me a good default value or idea of how can I choose ones?

Best,
Saeed

**Comment by njzjz** [comment, marked as answer]
As the unit of virial is as the same as that of energy, you can use the same value for virial as energy.
